File: binarysearch.js
function binarysearch(arr,key){
    s=0
    e=arr.length-1
    while(s<=e){
        mid=parseInt((s+e)/2)
        if(arr[mid]==key){
            return mid
        }else if(arr[mid]>key){
            e=mid-1
        }else{
            s=mid+1
        }
    }
return -1
}
let se=9
let arr=[1,9]
let result=binarysearch(arr,se)
if(result!=-1){
    console.log(`The Element is found at index ${result}`)
}
else{
    console.log(`The Element Not found!`)
}

File: binarysearch.py
def binarysearch(arr,key):
    start=0
    end=len(arr)-1
    while(start<=end):
        mid=(start+end)//2
        if(arr[mid]==key):
            return mid
        elif(arr[mid]>key):
            end=mid-1
        else:
            start=mid+1
    return-1
arr=list(map(int,input().split()))
search=int(input())
result=binarysearch(arr,search)
if(result!=-1):
    print(f"the Element is found at index {result}")
else:
    print("The Elemeny Not Found!")